ui/lib/kv/addon/routes/create.js

```diff
@@ -21,7 +21,7 @@ export default class KvSecretsCreateRoute extends Route {
     return hash({
       backend,
       path,
-      secret: this.store.createRecord('kv/data', { backend, path }),
+      secret: this.store.createRecord('kv/data', { backend, path, casVersion: 0 }),
```

Comment:
Manually add `casVersion: 0` here since it's a new secret

Reply:
is it worth adding a comment here similar to what we did in the `edit.js` route to point out this is set manually and reference the serializer for why? [github link for edit.js](https://github.com/hashicorp/vault/blob/9a70022eb90a96e05ec6f4bd4034f8cb3fcd9ac8/ui/lib/kv/addon/routes/secret/details/edit.js#L22)